docs/examples/index.rst

```diff
@@ -0,0 +1,118 @@
+*********
```

Comment:
the "Learn more" link on https://flax--2367.org.readthedocs.build/en/2367/ still points to https://flax--2367.org.readthedocs.build/en/2367/examples.html which is no 404